src/main/java/uk/gov/hmcts/reform/sandl/snlrules/drools/FactModification.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.sandl.snlrules.drools;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import uk.gov.hmcts.reform.sandl.snlrules.model.Fact;
+
+@Data
+public class FactModification {
+    private Fact oldFact;
+    private Fact newFact;
+
+    public FactModification(Fact oldFact, Fact newFact) {
```

Comment:
Is this needed? @AllArgsConstructor should work?

Reply:
true, not needed